UndertaleModTool/UndertaleModTool.csproj

```diff
@@ -99,6 +99,7 @@
         <PackageReference Include="System.Diagnostics.StackTrace">
             <Version>4.3.0</Version>
         </PackageReference>
+        <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
```

Comment:
Is it necessary?
Same for two lines below (115-116).

Reply:
I think I accidentally added those when testing the .NET upgrade, so I'm now removing them.